src/components/ModalVersion.ts

```diff
@@ -0,0 +1,16 @@
+import template from './ModalVersion.html';
+import * as ko from 'knockout';
+import client from '../api/client';
+
+class ModalVersion {
+  version: ko.Observable<string | null>;
```

Comment:
@DianaVargasOrellana I don't think we need to support the null value explicitly. Do we have a scenario for what we are using these two types?

Reply:
I changed this variable to accept only string type